modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-cluster-wide-migration_{context}"]
+= About cluster-wide migration
+
+During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other one is associated with the {SMProduct} 3.0 installation. You must carefully plan out the migration steps in order to avoid possible conflicts between the two control planes. The following topics describe the required conditions for successful migration.
+
+== Control planes and root certificate
+
+During the migration, the control plane associated with the 2.6 installation and the control plane associated with the 3.0 installation must share the same root certificate. This can be achieved by installing the 3.0 control plane to the same namespace as 2.6 control plane. The migration procedures show how to verify the root certificate is shared.
+
+== Control planes, discovery selectors, and namespace access
+
+Both control planes must have access to all namespaces in the mesh. During the migration, some proxies are controlled by the 3.0 control plane while other proxies remain controlled by the 2.6 control plane. To ensure that mesh communication works during the migration both control planes must be aware of the same set of services. Service discovery is provided by `istiod` component, which runs in the control plane namespace.
+
+In the {SMProduct} 3.0 installation, you can control how {istio} discovers services using discovery selectors. When using discovery selectors, ensure that the `discoverySelectors` expression defined in the {SMProduct} 3.0 {istio} resource match the namespaces that comprise the {SMProduct} 2.6 mesh. You might have to add additional labels to the {SMProduct} 2.6 application namespaces to ensure that they are captured in the {SMProduct} 3.0 installation. See "Scoping the service mesh with DiscoverySelectors" for more information.
+
+[NOTE]
+====
+In {SMProduct} 2.6 installations, the `maistra.io/member-of` label is automatically created. This label automatically gets removed during the migration process so it cannot be used.
```

Comment:
```suggestion
In {SMProduct} 2.6 installations, the `maistra.io/member-of` label is automatically created. This label cannot be used because it is automatically removed during the migration process.
```

Reply:
done